e2e/src/submission-finalization.spec.ts

```diff
@@ -152,21 +129,20 @@ describe("Submission and finalization test suite", () => {
   it.concurrent(
     "Check L2 safe/finalized tag update on sequencer",
     async () => {
-      const lineaRollup = config.getLineaRollupContract();
       const sequencerEndpoint = config.getSequencerEndpoint();
       if (!sequencerEndpoint) {
         console.log('Skipped the "Check L2 safe/finalized tag update on sequencer" test');
         return;
       }
 
-      const lastFinalizedL2BlockNumberOnL1 = (await getFinalizedL2BlockNumber(lineaRollup)).toString();
+      const lastFinalizedL2BlockNumberOnL1 = 0;
       console.log(`lastFinalizedL2BlockNumberOnL1=${lastFinalizedL2BlockNumberOnL1}`);
 
-      let safeL2BlockNumber = -1,
-        finalizedL2BlockNumber = -1;
+      let safeL2BlockNumber = 0,
+        finalizedL2BlockNumber = 0;
       while (
-        safeL2BlockNumber < parseInt(lastFinalizedL2BlockNumberOnL1) ||
-        finalizedL2BlockNumber < parseInt(lastFinalizedL2BlockNumberOnL1)
+        safeL2BlockNumber < lastFinalizedL2BlockNumberOnL1 ||
+        finalizedL2BlockNumber < lastFinalizedL2BlockNumberOnL1
```

Comment:
Can we revert the above changes? Cause essentially what we're doing here is just bypassing the while loop every time and check if `safeL2BlockNumber` and `safeL2BlockNumber` are equal to 0

Reply:
@jpnovais Any though on this ?